src/components/Dashboard/FirstSteps/Notification/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Notification = ({ iconClass, title, description }) => {
+  return (
+    <div className={`dp-step ${iconClass}`} role="alert" aria-label="notification">
+      <h4>
+        <span className="dp-iconstep" />
+        {title}
+      </h4>
+      <small dangerouslySetInnerHTML={{ __html: description }} />
```

Comment:
Is it fine to use a `small` element for the main content of the card?

Reply:
Justamente lo estaba platicando con @GusBaamonde en la mañana ... Ese `small` deberia cambiar a un `div`, ya que si de back llega un contenido tipo `<p>Lorem ipsun dolor sit amet</p>` causaria un warning porque un div no puede aparecer como descendiente de un `small`